app/main.py

```diff
@@ -1,8 +1,35 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
```

Comment:
Consider adding validation for the `clean_mark` attribute to ensure it is within a valid range (e.g., 0 to 100). This will prevent logical errors if invalid values are passed.